client/src/components/Footer.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <header class="bg-topography py-8 px-8 relative pin-b pin-l">
+    <div class="container mx-auto py-8 flex border-b border-grey-light text-left">
+      <div class="w-1/4 h-24">
+        <h1 class="font-bold text-xl">FOXYCLE</h1>
+      </div>
+      <div class="w-1/4 h-24">
+        <h1 class="font-bold text-sm leading-loose">HOURS</h1>
```

Comment:
We can make these all-caps with Tailwind instead of actually typing in all-caps.
```suggestion
        <h1 class="font-bold text-sm leading-loose uppercase">Hours</h1>
```